### PR TITLE
Deglitcher interface for Views

### PR DESCRIPTION
**Pull request by rpiaggio**

### CODE REVIEW

**Review by cquiroz — APPROVED** [review]
LGTM

**Comment by rpiaggio** [comment]
Now this ready for review

**Review by hugo-vrijswijk — COMMENTED** [review]
Any chance for some tests? :)

File: modules/tests/shared/src/test/scala/crystal/ViewFLawsSpec.scala [hunk 1]
```diff
@@ -3,7 +3,6 @@
 
 package crystal
 
-import cats.Eq
```
Comment (toddburnside) [inline modules/tests/shared/src/test/scala/crystal/ViewFLawsSpec.scala:6]: Heh. At first I read this as View Flaws Spec.

**Comment by rpiaggio** [comment]
> Now this ready for review

yup, working on that

File: modules/core/shared/src/main/scala/crystal/ViewThrottlerF.scala [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+  syncToAsync:   F ~> G,
+  dispatchAsync: G[Unit] => F[Unit]
+)(using G: Temporal[G]) {
+
+  private def curb: F[Unit] =
+    dispatchAsync:
+      G.monotonic.flatMap(now => waitUntil.set(now + timeout))
+
+  private def throttlerView(view: ViewF[F, A]): ViewF[F, A] =
+    view.withOnMod(_ => curb) // todo try to curb before modding?? maybe not necessary
```
Comment (toddburnside) [inline modules/core/shared/src/main/scala/crystal/ViewThrottlerF.scala:43]: The timeout is reset with each modification? Good.

**Review by toddburnside — APPROVED** [review]
This looks great to me! But, I agree that some tests would be nice. :)

**Comment by rpiaggio** [comment]
Tests were added, and the fixes for the bugs they revealed.

**Pull request merged**